docs/UserGuide.md

```diff
@@ -155,37 +155,37 @@ a good user experience. Excel sheets downloaded from LumiNUS have 2 rows preceed
 result from writing anything or adding information that does not abide by the standard format in the file.
 </div>
 
-#### Editing a person : `edit`
+#### Editing a student : `edit`
 
-Edits an existing person in the student roster.
+Edits an existing student in the student roster.
 
 Format: `edit INDEX [n/NAME] [a/STUDENT_NUMBER] [e/EMAIL] [m/MOD] [g/GROUP] [t/TAG]…​`
 
-* Edits the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index **must be a positive integer** 1, 2, 3, …​
+* Edits the student at the specified `INDEX`. The index refers to the index number shown in the displayed student list. The index **must be a positive integer** 1, 2, 3, …​
```

Comment:
I think it might be good to include some negative examples of what should not be done (ie edit 0, edit -100) etc

Reply:
You can reference what i did for deltask in my section!

Reply:
Its a good idea, will do!